tests/TestHILDO.py

```diff
@@ -0,0 +1,166 @@
+import json
+import time
+import random
+
+from drops.DropsDriver import myClient
+from drops.helpers.JsonFileHandler import JsonFileHandler
+from drops.helpers.ServerResponse import ServerResponse
+
+# pytest encourages this pattern, apologies.
+ip = "172.21.148.101"
+port = 9999
+supported_json = "drops/supported.json"
+client = myClient(ip=ip, port=port, supported_json=supported_json, reload=False)
+
+# create config parser handler
+json_handler = JsonFileHandler(supported_json)
+# load configs and launch web server
+json_handler.reload_endpoints()
+
+
+'''
+## Three classes of HIL tests
+1. Trivial, the supported.json has the key, value pair we expect to see on the device
+2. The device value does not equal the value associated with the key in supported.json (because its a live value) and we want to **schema match**
+3. We permutate the state of the device via actuation and we monitor the key value pair assocaited with the actuation to confirm it was successful
+'''
+
+
+def busy_wait(timeout: int):
```

Comment:
would it be meaningful to log when the busy wait fails?

In a future PR do we want to break this out into a helper function? Takes a lambda of what to wait on, in this case `r.STATUS['Status'] == "Busy"` and a timeout and logs which one caused the wait to exit?

Reply:
helper function that is not in this test file for clarity